### PR TITLE
Implement admin role assignment

### PR DESCRIPTION
**Pull request by timreimherr**
Same issue here. The files that I pushed are no longer in the branch. When I switch to this branch and open the solution it can't find the files that pushed to this branch (Role.cs, IRole.cs, RoleRepository.cs)

### CODE REVIEW

File: Blog/GamingGuruBlog.Data/Repositories/UserRepository.cs [hunk 1]
```diff
@@ -44,6 +44,12 @@ public void EditUser(User editedUser)
 
                 connection.Execute("Update AspNetUsers SET FirstName = @FirstName, LastName =@LastName Where ID = @UserID", parameters);
 
+                if (editedUser.Role != null)
+                {
+                    // need to get user Id then add role
```
Comment (wojonet) [inline Blog/GamingGuruBlog.Data/Repositories/UserRepository.cs:49]: ???

File: Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/CategoryServices.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GamingGuruBlog.Domain
+{
+    class CategoryServices
+    {
```
Comment (wojonet) [inline Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/CategoryServices.cs]: Also here?

**Comment by timreimherr** [comment]
Sorry, I was kind of tired mentally and just ended up creating the files
without implementing code. My idea for the User repo changes is set up for
adding a feature in the Admin panel. Currently, under the 'Users' tabs it
lists all the users that have registered with the web site. One of the
requirements was to give the owner the ability to assign roles to users so
that they could have other contributors to the blog. So, my idea is to
create a dropdown list next to each user that contains all the available
user roles and then use jQuery or Angular to send the userID and selected
value with the OnChange event, without refreshing the page. Now that I'm
working on it a little more, I'm not sure if this is where I should have
the changes saved.

Thoughts?

**Comment by wojonet** [comment]
Not really, that'll work. Only one role per user though...Leave it for now
make it multiple roles per user later.

**Comment by timreimherr** [comment]
Okay, so just focus on the category, tag, and user services for now?

**Comment by timreimherr** [comment]
When I'm in the TagServicesImplementation branch on git and open the
solution I get this:
[image: Inline image 1]


And when I'm in the Implement
[image: Inline image 2]


How do I fix this?

**Comment by timreimherr** [comment]
@wojonet 
This branch is not done because of my misuse of branches. Certain files were created while in another branch and then this branch couldn't find them. I understand branches a lot better now ha. So, once the other branches are merged I think I'll be able to use those files and finish this branch.

**Comment by wojonet** [comment]
@timreimherr you need to resolve all the merge conflicts in your PRs before I can finish reviewing.

**Comment by timreimherr** [comment]
@wojonet Ah, I see. Okay will do.

**Comment by timreimherr** [comment]
@wojonet these merge conflicts have been resolved

**Pull request merged**